dockstore-webservice/src/main/java/io/dockstore/webservice/core/docker/package-info.java

```diff
@@ -0,0 +1,23 @@
+/*
+ *    Copyright 2021 OICR
```

Comment:
I think Denis wanted to add UCSC here. I updated the Copyright text in the wiki somewhere